### PR TITLE
Add github release notes to justification

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Fixes: #1819 

## This introduces a breaking change

- [x] No

### CODE REVIEW

**Review by harshad16 — COMMENTED** [review]
lgtm
/lgtm

File: docs/source/prescription.rst [hunk 1]
```diff
@@ -1615,3 +1610,47 @@ Suggested changes to the manifest files used for deployment.
...
+GitHubReleaseNotesWrap
+======================
+
+A specific type of wrap pipeline unit that adds links to GitHub release page.
+
+GitHubReleaseNotesWrap ``run.release_notes``
+############################################
+
+A list of entries describing release for which resolved a link
+a link to GitHub release should be constructed.
```
Comment (pacospace) [inline docs/source/prescription.rst:1623]: `a link`

Comment (fridex) [reply docs/source/prescription.rst]: Adjusted in d292f95b, thanks for spotting this 👍🏻

**Pull request merged**